pristine-zen-garden.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" media="screen" href="/214/214.css?v=8may2013">
+  <link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Dave Shea">
+  <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+  <meta name="robots" content="all">
+  <title>CSS Zen Garden: The Beauty of CSS Design</title>
+</head>
+
+<body id="css-zen-garden">
+  <div class="page-wrapper">
+    <section class="intro" id="zen-intro">
+      <header role="banner">
+        <h1>CSS Zen Garden</h1>
+        <h2>
```

Comment:
@simon-jk-casey Hi! This code looks very nice (to my standards, he he). The only comment is here you indented a header here. "Note: We don't indent <h1> & other headers because they're short, but if you want to indent <h#>, that would be fine." From here https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/. So basically not really any comments